lib/runtime/wasmer/test_helpers.go

```diff
@@ -26,16 +26,18 @@ func NewTestInstance(t *testing.T, targetRuntime string) *Instance {
 	return NewTestInstanceWithTrie(t, targetRuntime, nil)
 }
 
-// NewTestInstanceWithTrie will create a new runtime (polkadot/test) with the supplied trie as the storage
+// NewTestInstanceWithTrie will return a wasmer.Instance given the targetRuntime, using the provided trie as the storage
+// the targetRuntime can be a file path to a local .wasm file or a already defined constant in runtime/constants.go
```

Comment:
supernit no need to specify `wasmer.Instance` (check the function signature) + hopefully simplify/clarify a tiny bit


```suggestion
// NewTestInstanceWithTrie returns an instance based on the target runtime string specified, 
// which can be a file path or a constant from the constants defined in `lib/runtime/constants.go`.
// The instance uses the trie given as argument for its storage.
```